util/fbrun/FbRun.cc

```diff
@@ -414,18 +414,19 @@ void FbRun::tabComplete(const std::vector<std::string> &list, int &currentItem,
         ++split; // skip space
     std::string prefix = text().substr(split, m_completion_pos - split);
 
+    int listSize = list.size();
```

Comment:
looks like `list.size()` is actually `unsigned` (`size_t`)

http://www.cplusplus.com/reference/vector/vector/size/